dts/bindings/sensor/invensense,icm4268x-common.yaml

```diff
@@ -0,0 +1,368 @@
+    type: int
+    default: 0
+    description: |
+      Choose if the interrupt will be automatically latched and then cleared (pulsed mode)
+      Or latch and clear happens only when user code requests the data (latched mode).
+      Default interrupt mode is pulsed.
+
+      0 # ICM4268X_DT_INT1_MODE_PULSED
+      1 # ICM4268X_DT_INT1_MODE_LATCHED
+
+    enum: [0, 1]
+
+  fifo-mode:
+    type: int
+    default: 0
+    description: |
+      Select what to do when FIFO is full. Stream to FIFO mode will overwrite LRU frame.
+      Stop on Full mode will prevent writing any frame if both FIFO and cache are full.
+      Default fifo mode is bypass.
+
+      0 # ICM4268X_DT_FIFO_MODE_BYPASS
+      1 # ICM4268X_DT_FIFO_MODE_STREAM
+      3 # ICM4268X_DT_FIFO_MODE_STOP_ON_FULL
+
+    enum: [0, 1, 3]
+
+  fifo-ths-int-clr:
+    type: int
+    default: 1
+    description: |
+      Fifo threshold Interrupt Clearing Options.
+      Default fifo threshold interrupt (in latched mode) gets cleared on status bit read.
+
+      1 # ICM4268X_DT_FIFO_THS_INT_CLR_STSRD
+      2 # ICM4268X_DT_FIFO_THS_INT_CLR_DATARD
+      3 # ICM4268X_DT_FIFO_THS_INT_CLR_BOTH
+
+    enum: [1, 2, 3]
+
+  fifo-full-int-clr:
+    type: int
+    default: 1
+    description: |
+      Fifo Full Interrupt Clearing Options.
+      Default fifo full interrupt (in latched mode) gets cleared on status bit read.
+
+      1 # ICM4268X_DT_FIFO_FULL_INT_CLR_STSRD
+      2 # ICM4268X_DT_FIFO_FULL_INT_CLR_DATARD
+      3 # ICM4268X_DT_FIFO_FULL_INT_CLR_BOTH
+
+    enum: [1, 2, 3]
+
+  tmst-en:
+    type: boolean
+    description: |
+      Enable or disable the timestamp register.
+      Timestamp register is enabled by default.
+
+  tmst-fsync-en:
+    type: boolean
+    description: |
+      Enable/disable timestamp fsync register.
+      Default timestamp feature of FSYNC is enabled.
+
+  tmst-delta-en:
+    type: boolean
+    description: |
+      Timestamp register will have absolute or delta values from the last occurrence of ODR.
+
+  tmst-res:
+    type: int
+    default: 0
+    description: |
+      If 0, Timestamp Resolution is 1us, if 1 and RTC is enabled then time res is
+      RTC clock period else 16us.
+      Default timestamp resolution is set to 1us.
+
+      0 # ICM4268X_DT_TMST_RES_1US
+      1 # ICM4268X_DT_TMST_RES_RTC
+
+    enum: [0, 1]
+
+  tmst-regs-en:
+    type: boolean
+    description: |
+      If false, TMST_VALUE returns zeros else returns timestamp values.
+
+  accel-pwr-mode:
+    type: int
+    default: 0
+    description: |
+      Specify the accelerometer power mode.
+      Default power mode for accelerometer is power-up.
+
+      0 # ICM4268X_DT_ACCEL_OFF
+      2 # ICM4268X_DT_ACCEL_LP
+      3 # ICM4268X_DT_ACCEL_LN
+
+    enum: [0, 2, 3]
+
+  accel-odr:
+    type: int
+    default: 6
+    description: |
+      Specify the accelerometer output data rate expressed in samples per second (Hz).
+      Default accelerometer ODR is 1kHz in Low Noise Mode.
```

Comment:
I feel like there is still a misunderstanding here about the default values, please read this link https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values, you need to say *why* something is the default value

Reply:
The reason for choosing a default value is not mentioned in the Invensense datasheet. 
Though I can think of a logical reason myself.. but is it recommended to write something on behalf of the part owner ?

Reply:
> The reason for choosing a default value is not mentioned in the Invensense datasheet. Though I can think of a logical reason myself.. but is it recommended to write something on behalf of the part owner ? Also this link for Zephyr default rules has an example where it says - "The default was recommended in the manufacturer datasheet"
> 
> So if you recommend, I can write like this - "The default value for accelerometer ODR as recommended in the manufacturer datasheet is 1kHz in Low Noise Mode"

that's fine to write that

Reply:
Done.. please check